salsa_webqa/salsa_runner.py

```diff
@@ -156,14 +173,26 @@ def run_on_browserstack(self):
 
     def run_locally(self):
         """ Runs tests on local browser """
-        self.cleanup_results()
         print('Running for browser: ' + self.driver_name)
         return self.trigger_pytest(self.driver_name)
 
+    def run_non_selenium(self):
+        """ Runs non_selenium_tests"""
+        print('Running non selenium tests')
+        return self.trigger_pytest(None)
+
     def trigger_pytest(self, config_section):
         """ Runs PyTest runner on specific configuration """
+        if config_section is None or self.reporting == 'simple':
+            path = os.path.join(self.project_root, 'non_selenium_tests')
+            if not os.path.exists(path):
+                print("Can't run non selenium tests files are not found")
+                return None
+            pytest_arguments = [os.path.join(self.project_root, 'non_selenium_tests')]
+            pytest_arguments.append('--junitxml=' + os.path.join(self.result_folder, "Non_selenium_report" + '.xml'))
+            pytest_arguments.append('--html=' + os.path.join(self.result_folder, "Non_selenium_report" + '.html'))
```

Comment:
You can rewrite those three lines so you declare `pytest_arguments` list just once (instead of declaring list and then appending to it twice).
PyCharm can do this for you automatically (It even warned me of this issue :)).